src/main/java/io/github/enkarin/bookcrossing/admin/service/AdminService.java

```diff
@@ -19,7 +19,9 @@
 public class AdminService {
 
     private final UserRepository userRepository;
+
```

Comment:
These empty lines look redundant. I'd recommend to exclude corresponding checkstyle rule